docs/consensus/keymanager.md

```diff
@@ -0,0 +1,39 @@
+# Key Manager
+
+The key manager service is responsible for coordinating the SGX-based key
+manager runtimes. It stores and publishes policy documents and status updates
```

Comment:
At a minimum we should say what policies cover: which enclaves are allowed to fetch which keys, and which enclaves the master secret can be replicated to?

Reply:
Added.